### PR TITLE
Balancing vehicle skills across USCM

### PR DESCRIPTION
**Pull request by majchroo**
# About the pull request

**tl;dr; Add VAN driving skills to many USCM roles. Add ARC skills to CE**

Current state of VAN skills:

> General, CO, XO, ASO, SO, SL, Combat Engineer

New state of VAN skills:

> General, CO, XO, ASO, SO, SL, Combat Engineer, **Spec FTL, IO, DP, DCC, MP, MW, MT, CT, CMP, RO**

Current state of ARC skills:
> General, CO, XO, SO, SEA

New state of ARC skills:
> General, CO, XO, SO, SEA, **CE**

# Explain why it's good for the game

IO's - It's logical for mobile officers working alone to have driving skills
CE+MT - they can fix em but can't ride em as of now
Spec, FTL - because they often go aSL and have wide range of skills
DP, DCC - this is basically to allow them to move small vehicles out of their dropships
MP,MW,CMP - military police cant handle small transport vehicles as of now
RO, CT - logistics cant handle small small transport vehicles as of now

Further forum discussion: https://forum.cm-ss13.com/t/balancing-vehicle-skills-across-uscm/11688/15

# Testing Photographs and Procedure
Spawned Spec and CT. Can confirm that they can drive VAN after the change.
<details>
<summary>Screenshots & Videos</summary>

Screenshot

</details>

# Changelog

:cl:Adamix147
balance: Grant VAN vehicle skills across USCM. Grant ARC skills to CE.

/:cl:

### CODE REVIEW

**Comment by Red-byte3D** [comment]
https://github.com/cmss13-devs/cmss13/pull/5401 pretty sure @Zonespace27 was against giving anything but actual CIC command training for the arc

**Comment by Somniworld** [comment]
> #5401 pretty sure @Zonespace27 was against giving anything but actual CIC command training for the arc

Hiya, is that why IOs can't drive the ARC? I understood that it was meant to serve as a drop-off point for the intel team, which is  currently very difficult to do when only SOs reliably ever drive them. 

Perhaps replacing the CE with IOs would be more thematic ? Especially since they get van driving skills in this very PR.

**Comment by majchroo** [comment]
> Hiya, is that why IOs can't drive the ARC? I understood that it was meant to serve as a drop-off point for the intel team, which is currently very difficult to do when only SOs reliably ever drive them.

@Somniworld I found this while working on this PR:
https://forum.cm-ss13.com/t/give-ios-the-ability-to-drive-the-arc/9295

> Not entirely sold on the benefit of having CEs be able to drive the ARC. (Having IOs do it would be worse so don't do that)

@Zonespace27 Let me know if you want it removed from the PR.

I wont push it because it serves no other gameplay purpose than reparking it in FOB. CE can't really leave FOB/comms because of SOP anyway.

I did add it because:
1) Reparking 
2) It was voiced on forum that it's logical for master of engineering to have driving skills.

**Comment by HeresKozmos** [comment]
IOs shouldn't have driving skill, them picking up a vehicle and driving around the colony negates the biggest threat to them which is them having to travel alone/in small groups into the sideline/backline, and it will also let them just fill the vehicle with giant piles of intel basically negating any reason to travel back and forth unless they just stock up giant piles of it.

**Comment by DorfGuy1** [comment]
Add it to medics too. Marine ambulance. Ugh. I've been saying for months that marines should get a truck by default as a correspondence to the mobility that tunnels offer.

**Comment by Ammoniacres** [comment]
> IOs shouldn't have driving skill, them picking up a vehicle and driving around the colony negates the biggest threat to them which is them having to travel alone/in small groups into the sideline/backline, and it will also let them just fill the vehicle with giant piles of intel basically negating any reason to travel back and forth unless they just stock up giant piles of it.


IOs having driving skills would be incredibly fun, but negating the danger of travelling should come with some sincere risks or a cost beyond finding wheels. It also shouldn't be great for carting intel.
